Первая версия персональной страницы

mrslander opened the pull request:
Первая версия

MartyDashP committed:
Первая версия персональной страницы

Mrak2017 reviewed (commented):
1) В плане кода сделано красиво, но переусложнено. В рамках первой работы необходимо было ограничиться только html и css. Никаких Js, Scss и package.json. Все это будет мешать в следующей домашней работе.
Соответственно необходимо всё это убрать, и оставить только html и css
2) На странице "досье" сбилось форматирование, т.к. в блоке текста переносы строки идут без использования тега <br>
3) Явно не указана кодировка страницы, из за чего, в разных браузерах она может отображаться некорректно (наблюдал это в IE)

MartyDashP committed:
- Проект избавлен от лишних файлов
- На странице указана кодировка а так же внесен тег говорящий интерпритировать страницу по спецификации HTML5
- В блоке "Досье" изменена структура текста и ширина блока для поддержки IE11 (чертов IE)
- Блок навигации изменил тип позиционирования для поддержки IE11 (чертов IE)

MartyDashP committed:
- Удален лишний тег </p>

Mrak2017 commented:
Теперь всё хорошо, можно мержить
